src/main/java/io/strimzi/kafka/bridge/http/HttpConfig.java

```diff
@@ -29,6 +29,9 @@ public class HttpConfig extends AbstractConfig {
     public static final String DEFAULT_HOST = "0.0.0.0";
     public static final int DEFAULT_PORT = 8080;
     public static final long DEFAULT_CONSUMER_TIMEOUT = -1L;
+    public static final String HTTP_CONSUMER_ENABLED = HTTP_CONFIG_PREFIX + "consumer.enabled";
+
+    public static final String HTTP_PRODUCER_ENABLED = HTTP_CONFIG_PREFIX + "producer.enabled";
```

Comment:
I would move these constants in the previous section where we already have all the `HTTP_` prefixed constants.